### DISCUSSION TITLE
any freelance ? , HELP turnstile(CF) click using nodriver

### DISCUSSION THREAD

**Discussion by addame2** [ultrafunkamsterdam/undetected-chromedriver#1950] [Q&A]
<app-cloudflare-captcha-container class="ng-star-inserted"><div appcloudflarerecaptcha="" class="my-10 ng-star-inserted"><div class="cf-turnstile-wrapper" style="border: 0px; margin: 0px; padding: 0px;"><input type="hidden" name="cf-turnstile-response" id="cf-chl-widget-hxskw_response"></div></div></app-cloudflare-captcha-container>

anyone cant help how I can click this after click if get sucess then I will click button if not will try again 

<img width="1187" alt="Screenshot 2024-07-17 at 09 37 26" src="https://github.com/user-attachments/assets/7507bcff-17bb-4281-bcbb-663ce9d975ee">

**Comment by devblack** [comment]
Contact me
